changes-set.txt

```diff
@@ -21,6 +21,28 @@ proposed  sylbird   biimtrrd    etc.
 proposed  syl5*     *trid       (syl5bi -> biimtrid; syl5eqel -> eqeltrid;etc.)
 proposed  syl6*     *trdi
 proposed  *mpt2*    *mpo*       (renaming process underway)
+proposed  cnmpt2k   cnmpompk
+proposed  cnmptk2   cnmpkmpo
+proposed  cnmptk1p  cnmpkmptp
+proposed  cnmptkk   cnmpkmpk
+proposed  cnmpt1k   cnmptmpk
+proposed  cnmptk1   cnmpkmpt
+proposed  cnmptcom  cnmpocom
+proposed  cnmpt2res cnmpores
+proposed  cnmpt1res cnmptres
+proposed  cnmpt22f  cnmpompof
+proposed  cnmpt22   cnmpompo
+proposed  cnmpt2t   cnmpot
+proposed  cnmpt21f  cnmpomptf
+proposed  cnmpt21   cnmpompt
+proposed  cnmpt2c   cnmpoc
+proposed  cnmpt2nd  cnmpo2nd
+proposed  cnmpt1st  cnmpo1st
+proposed  cnmpt12   cnmptmpo
+proposed  cnmpt12f  cnmptmpof
+proposed  cnmpt1t   cnmptt
+proposed  cnmpt11f  cnmptmptf
+proposed  cnmpt11   cnmptmpt
```

Comment:
```suggestion
proposed  cnmpt2k   cnmpok
proposed  cnmptk2   cnmpko
proposed  cnmptk1p  cnmpktp
proposed  cnmptkk   cnmpkk
proposed  cnmpt1k   cnmptk
proposed  cnmptk1   cnmpkt
proposed  cnmptcom  cnmpocom
proposed  cnmpt2res cnmpores
proposed  cnmpt1res cnmptres
proposed  cnmpt22f  cnmpoof
proposed  cnmpt22   cnmpoo
proposed  cnmpt2t   cnmpoot
proposed  cnmpt21f  cnmpotf
proposed  cnmpt21   cnmpot
proposed  cnmpt2c   cnmpoc
proposed  cnmpt2nd  cnmpo2nd
proposed  cnmpt1st  cnmpo1st
proposed  cnmpt12   cnmpto
proposed  cnmpt12f  cnmptof
proposed  cnmpt1t   cnmpttt
proposed  cnmpt11f  cnmpttf
proposed  cnmpt11   cnmptt
```
another potential idea that preserves the merging of the "prefix" mp
it's not as preserving though, since cnmpt11 collides with cnmpt1t (and cnmpt21 with cnmpt2t)

approve either way, I'm not an expert in this section